data/json/terrain.json

```diff
@@ -332,6 +348,7 @@
         "type" : "terrain",
         "id" : "t_clay",
         "name": "clay",
+        "description": "A field full of malleable clay, suitable for kiln firing if it was extracted properly.",
```

Comment:
Likewise, `extracted properly` sounds like a mystery. Does this just require a shovel?

Reply:
Without usurping player's character intelligence (AKA "how much would any character be able to recognize this?" like comparing a character with 4 intelligence to one with 12 or one as a churl another as a MD), it's more of a hint to players to investigate how to extract things